util/config/config.go

```diff
@@ -45,16 +47,19 @@ const (
 	DonationAdjustDivisorFactor  = 2
 	MinGenIDRegistrationFee      = 0
 	GenerateIDBlockDelay         = 1
-	RandomBeaconLength           = 32
+	RandomBeaconUniqueLength     = vrf.Size
+	RandomBeaconLength           = vrf.Size + vrf.ProofSize
 	ProtocolVersion              = 71
 	MinCompatibleProtocolVersion = 71
 	MaxCompatibleProtocolVersion = 75
 	DefaultTxPoolCap             = 32
+	ShortHashSize                = uint32(8)
 )
 
 var (
-	ShortHashSize uint32 = 8
-	ShortHashSalt []byte = util.RandomBytes(32)
+	ShortHashSalt    = util.RandomBytes(32)
+	GenesisTimestamp = time.Date(2018, time.January, 0, 0, 0, 0, 0, time.UTC).Unix()
```

Comment:
@yilunzhang 
fmt.Println(time.Date(2018, time.January, 0, 0, 0, 0, 0, time.UTC))
`2017-12-31 00:00:00 +0000 UTC`
Is it your expectation?

How about `2018-05-28`(first listed by exchange) or `2018-03-05`(first commit forked from OnChain DNA)?

Reply:
I was thinking maybe we should change this right before we launch mainnet so the timestamp of block 0 and 1 are close